packages/docusaurus-theme-openapi/src/theme/ApiPage/styles.module.css

```diff
@@ -46,6 +46,30 @@
   --openapi-dropzone-hover-shim: rgba(235, 237, 240, 0.8);
 
   --openapi-inverse-color: var(--ifm-color-gray-100);
+
+  --openapi-api-method-get-background: #0e9b71;
+  --openapi-api-method-get-shadow: 1px 1px 0 #0d8d67, 0 1px 0 #0d8d67,
+    1px 0 0 #0d8d67;
```

Comment:
I would prefer to avoid adding new colors that don’t exist in Docusaurus so that colors will continue to mess if their theme changes

Reply:
Sounds good, I can replace the static HEX values to use the available Docusaurus CSS variables from: https://infima.dev/docs/utilities/colors.

|Method|Infima|
|---|---|
|`GET`| `var(--ifm-color-success)`|
|`POST`| `var(--ifm-color-primary)`|
|`PUT`|`var(--ifm-color-warning)`|
|`DELETE`|`var(--ifm-color-danger)`|
|`HEAD`|`var(--ifm-color-secondary)`|
|`PATCH`|`var(--ifm-color-info)`|

and update the documentation on how to customize the appearance further through the available CSS variables.

Reply:
Sounds good